### PR TITLE
Windows build fixes

### PR DESCRIPTION
**Pull request by MonsieurNicolas**

### CODE REVIEW

File: src/ledger/LedgerTxn.cpp [hunk 1]
```diff
@@ -2397,7 +2397,8 @@ LedgerTxnRoot::Impl::getBestOffer(Asset const& buying, Asset const& selling,
         prefetch(toPrefetch);
 
         cached->allLoaded =
-            std::distance(newOfferIter, offers.cend()) < BATCH_SIZE;
+            static_cast<size_t>(std::distance(newOfferIter, offers.cend())) <
```
Comment (jonjove) [inline src/ledger/LedgerTxn.cpp]: Exactly the same thing appears on line 2310. Is there no warning there?

Comment (MonsieurNicolas) [reply src/ledger/LedgerTxn.cpp]: interestingly it wasn't... regardless, I fixed the other call site to be consistent

**Comment by MonsieurNicolas** [comment]
@jonjove updated

**Comment by jonjove** [comment]
r+ abcfe2c

**Pull request merged**